classes/Tools.php

```diff
@@ -275,7 +275,12 @@ public static function getHttpHost($http = false, $entities = false, $ignore_por
             $httpHost = $_SERVER['HTTP_HOST'];
         }
 
-        $host = (isset($_SERVER['HTTP_X_FORWARDED_HOST']) ? $_SERVER['HTTP_X_FORWARDED_HOST'] : $httpHost);
+        if (isset($_SERVER['HTTP_X_FORWARDED_HOST'])) {
+            $host = trim(array_pop(explode(',', $_SERVER['HTTP_X_FORWARDED_HOST'])));
```

Comment:
why array_pop instead of the first one?

Reply:
And why explode with `,`? Because there is no RFC about having multiple values in this field.